### PR TITLE
Add Python RLI for mecanum drive

### PR DESCRIPTION
**Pull request by virtuald**

### CODE REVIEW

**Comment by sciencewhiz** [comment]
@Starlight220 should there be updates to inspector to handle other URLs?

**Comment by Starlight220** [comment]
It'll need a completely separate config JSON and everything.
I wonder how it'll be best to set things up, because merging the reports won't work with the current version of Inspector.

I can try to work it into Inspector, but that would likely take more time. I don't have a preference between upgrading Inspector for it vs adding the additional JSON and everything here, your choice.

File: source/docs/software/hardware-apis/motors/wpi-drive-classes.rst [hunk 1]
```diff
@@ -321,6 +321,14 @@ MecanumDrive is a method provided for the control of holonomic drivetrains with
       :linenos:
       :lineno-start: 31
 
+  .. tab:: Python
+
+    .. remoteliteralinclude:: https://raw.githubusercontent.com/robotpy/examples/main/mecanum-drive/robot.py
```
Comment (Starlight220) [inline source/docs/software/hardware-apis/motors/wpi-drive-classes.rst:326]: Regardless of Inspector, this should use a pinned version rather than `main`.

**Review by Starlight220 — CHANGES_REQUESTED** [review]

File: source/docs/software/hardware-apis/motors/wpi-drive-classes.rst (hunk 1 above)
Comment (virtuald) [reply source/docs/software/hardware-apis/motors/wpi-drive-classes.rst:326]: We don't have versions for our examples? But I suppose we can do that. What version do you want?

Comment (Starlight220) [reply source/docs/software/hardware-apis/motors/wpi-drive-classes.rst:326]: Like the allwpilib examples, it should be a git tag or commit hash.

Comment (sciencewhiz) [reply source/docs/software/hardware-apis/motors/wpi-drive-classes.rst:326]: The issue is that examples may be updated and if lines are added or removed the frc-docs RLI will point to junk.

**Comment by Starlight220** [comment]
After a rebase, the `inspector.json` will need to be updated with another object in the `sources` array.

**Comment by Daltz333** [comment]
Just a ping.

**Comment by virtuald** [comment]
Added a tag.

**Review by Daltz333 — APPROVED** [review]

**Pull request merged**